juju/juju
# Remove RemoteEntityId struct and just use token

## Description of change

This PR removes the unnecessary RemoteEntityId struct used with cmr and instead uses just the token in messages exchanged between models. The model uuid is stored with each remote application entity so is not neeed elsewhere. The code is simpler now.

## QA steps

bootstrap and relate mediawiki and mysql in different models.
remove relation and check status
add relation again and verify
